examples/native-external-references-mockshop/README.md

```diff
@@ -0,0 +1,312 @@
+Native external references make it possible to connect any how third-party systems link to existing reference fields.
+
+For the purpose of this example, we will be connecting to the [MockShop](https://mock.shop/) external system and resolving its data through Contentful GraphQL API.
+
+With Native external references we introduce the following new entity types that allow us to model the data from third-party systems in Contentful:
+
+* `Resource Provider` - a third-party system that provides resources. Each provider can have multiple resource types. In our example: a `MockShop` provider.
+* `Resource Type` - a specific type of resource (not the resource itself) that is provided by a resource provider. In our example: `Product`.
+# Instructions to create and run the app
+
+## Copying the example
+
+To get started with your application, you need to make a local copy of the example code by running the following command in your terminal. 
+
+> **NOTE:** 
+> Make sure you replace `<name-of-your-app>` with the name of your choice.
+
+```bash
+npx create-contentful-app@latest <name-of-your-app> --example native-external-references-mockshop
+```
+
+Once the process finishes, navigate to the directory that was created by running this command:
+
+```bash
+cd <name-of-your-app>
+```
+
+To complete the process, it is necessary to install all dependencies of the project by executing the following command:
+
+```bash
+npm i
+```
+
+## Creating a custom app definition
+
+To see your app within Contentful, you must first set it up. To do that, we will create an app definition, which is an entity that represents an app in Contentful and stores general information about it.
+
+To create the app definition, run this command:
+
+```bash
+npm run create-app-definition
+```
+
+You will need to answer the following questions in the terminal when prompted. Feel free to proceed with the default options provided.
+
+1. **Name of your application**. This is how your app will be named and it will be displayed in a few places throughout the UI. The default is the name of the folder you created.
+2. **Select where your app can be rendered**. This shows potential [app locations](https://www.contentful.com/developers/docs/extensibility/app-framework/locations/) where an app can be rendered within the Contentful Web app. Select **App configuration screen** , as we will utilize the configuration screen to provide the external API URL for the app.
+3. **Contentful CMA endpoint URL**. This refers to the URL used to interact with Contentful's Management APIs.
+4. **App Parameters**. These are configurable values that can be used to set default values or define custom validation rules. We need to define these for the API URL in this case:
+
+* Opt for **Y** to advance with the process of defining the parameters
+* Choose **Installation**
+* Input `MockShop API Endpoint` as **Parameter name** and `apiEndpoint` as **ID**
+* Select **Symbol** as type and mark the parameter as required
```

Comment:
I feel like we could better let the user opt for `n` and skip this as it might add more ways for the user to not complete the goal of the tutorial. App parameters are described elsewhere

Reply:
I can remove this but then we don't need this function `getMockShopUrl` anymore since this was getting the  `apiEndpoint` from  `context.appInstallationParameters`

Reply:
Yea, we're probably heading in the direction of removing the configuration screen. But I was referring to the fact that you can have a configuration screen without setting up the Parameters. It would still work that way.